client/src/components/EventView/index.js

```diff
@@ -1,7 +1,174 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import AppBar from "../Common/AppBar";
+import { eventPink, theme1, spacing64, fontSize24 } from "../../resources/style-constants";
+import Invitee from "./Invitee/index.js";
+import Map from "../Common/Map/index.js"
+import styled from "styled-components";
+import Button from "../Common/Buttons/SecondaryButton/index.js";
+import RSVPFloating from "../EventView/RSVPFloating/index.js";
+import ClientService from "../../services";
+import CenteredLoadingSpinner from "../Common/CenteredLoadingSpinner";
+import Stack from "../Common/Stack";
+import Error from "../Common/Error";
+//import PageAccess from "../Common/PageAccess";
 
-const EventView = () => {
-  return <div>Event View</div>;
+// const testEvent = {
+//     eventID: "testID",
+
+// }
+
+const CenterHeading = styled.p`
+    font-size: calc(10px + 2vmin);
+    color: black;
+    font-weight: bold;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+`;
+
+const CenterHeadingLight = styled.p`
+    font-size: calc(7px + 2vmin);
+    color: black;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+`;
+
+const Heading = styled.p`
```

Comment:
For headings, you should use h1. See the components inside of the EventEdit for an example of the headings. You'll want to match that styling/format so our pages look consistent.